### PR TITLE
feat(aiohttp): add support for aiohttp proxy config using env vars

### PR DESCRIPTION
**Pull request by icamposrivera**

### CODE REVIEW

File: kubernetes_asyncio/client/rest.py [hunk 1]
```diff
@@ -78,7 +78,8 @@ def __init__(self, configuration, pools_size=4, maxsize=None):
 
         # https pool manager
         self.pool_manager = aiohttp.ClientSession(
-            connector=connector
+            connector=connector,
+            trust_env=True
```
Comment (tomplus) [inline kubernetes_asyncio/client/rest.py:82]: This file is generated by https://github.com/OpenAPITools/openapi-generator and you have to modify the generator first.

Comment (icamposrivera) [reply kubernetes_asyncio/client/rest.py:82]: ok, I did it yesterday, let's see what happens :) thanks

https://github.com/OpenAPITools/openapi-generator/pull/11171/files

Comment (icamposrivera) [reply kubernetes_asyncio/client/rest.py:82]: the PR on openapi-generator was approved, sould this PR be approved also? Thanks!

**Comment by jjbarcas** [comment]
is there any update on this PR?

**Comment by tomplus** [comment]
I'll work on it soon. It's not very easy, I have to switch from Openapi-Generator v4.3.1 to ~v5.4.0 to get your change. I expect some extra time for testing and tuning may be needed.

**Pull request merged**